code/modules/mob/inventory.dm

```diff
@@ -236,6 +243,11 @@ var/list/slot_equipment_priority = list( \
 /mob/proc/unEquip(obj/item/I, var/atom/target)
 	if(!canUnEquip(I))
 		return
+	if(I && config.atom_content_limit && target)
+		var/turf/T = get_turf(target)
+		if(T.get_all_contents_count(list(/obj/structure, /obj/item)) >= config.atom_content_limit)
+			to_chat(src, SPAN_WARNING("There is no room to put that there."))
+			return 0
```

Comment:
This is almost straight copy-paste of https://github.com/NebulaSS13/Nebula/pull/353/files#diff-b0373f944c6d476ad32d44bf2693e238R186-R190

Reply:
Shouldn't this check be in `canUnEquip()`?

Reply:
Felt bleedy. canUnEquip is called by other stuff. Could have side-effects.